README.md

```diff
@@ -97,18 +94,18 @@ Additionally to the information provided on your application, note that your pro
 
 4. **Follow-up**
    - Once work on the project has started, the Web3 Foundation will want to have follow-up conversations to see how the project is progressing.
-   - To help speed up the milestone evaluation process, take a look at the [Guidelines for Milestone Deliverables](./grants/milestone-deliverables-guidelines.md)
+   - To help speed up the milestone evaluation process, take a look at the [Guidelines for Milestone Deliverables](./grants/milestone-deliverables-guidelines.md).
 
-## :mailbox_with_mail: RFP Suggestion
+## :mailbox_with_mail: RFP Suggestions
```

Comment:
I'm in favor of not using the abbreviation in the title. 

`## :mailbox_with_mail: Request for Proposal (RFP) Suggestions`

Reply:
```suggestion
## :mailbox_with_mail: Request for Proposal (RFP) Suggestions
```